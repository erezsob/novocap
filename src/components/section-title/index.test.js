import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Title from './index';

describe('<Title />', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Title title="test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('renders with a given title', () => {
    const wrapper = shallow(<Title title="AwesomeTitle" />);
    expect(wrapper.text()).toBe('AwesomeTitle');
  });
});
