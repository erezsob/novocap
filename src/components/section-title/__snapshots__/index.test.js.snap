// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> renders correctly 1`] = `
<div
  className="title"
>
  test
</div>
`;
